server/sonar-server/src/test/java/org/sonar/server/measure/persistence/MeasureDaoTest.java

```diff
@@ -37,13 +37,12 @@
 public class MeasureDaoTest extends AbstractDaoTestCase {
 
   DbSession session;
-
-  MeasureDao dao;
+  MeasureDao sut;
```

Comment:
what means sut ? On my side, using "dao" or "measureDao" makes more sens and is more easily understandable by everybody